docs/adguard-for-android/solving-problems/compatibility-issues.md

```diff
@@ -0,0 +1,39 @@
+---
+title: Known compatibility issues with Android apps
+sidebar_position: 14
+---
+
+## Conflict apps
+
+### Various VPN apps
+
+All VPN apps are incompatible with AdGuard running in VPN filtering mode since you cannot have two active VPNs at the same time. To solve this problem we can suggest you:
+
+* to use AdGuard VPN that has a compatibility mode with AdGuard out-of-the-box
+* to use AdGuard in Local HTTP Proxy mode. *Please note that 'manual' HTTP Proxy mode is broken since Android 10*
+* to figure out if the third-party VPN app supports SOCKS4/SOCKS5 mode. In this case you can configure a VPN app to work as outbound proxy and to set up local-host outbound proxy with params from the third-party app. AdGuard can be automatically configured with 'Orbot: Proxy with Tor'
```

Comment:
* if you use a third-party VPN app, figure out whether it supports SOCKS4/SOCKS5 mode. If it does, you can configure your VPN app to work as outbound proxy and set up local outbound proxy with params from the third-party app. For instance, AdGuard can be automatically configured with 'Orbot: Proxy with Tor'